plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java

```diff
@@ -80,7 +137,7 @@ public void testInsertInPresenceOfNotSupportedColumn()
     }
 
     @Test
-    public void testView()
+    public void testSimpleView()
```

Comment:
Can this test be considered superseded by `testView` from the superclass?

Reply:
Not this one, this creates a view in SqlServer but testView in the superclass creates it in Trino.